cross/go2rtc/Makefile

```
@@ -0,0 +1,31 @@
+PKG_NAME = go2rtc
+PKG_VERS = 1.6.2
+PKG_EXT = tar.gz
+PKG_DIST_SITE = https://github.com/AlexxIT/go2rtc/archive/refs/tags/v$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_FILE = v$(PKG_VERS).$(PKG_EXT)
```

There must be PKG_DIST_FILE that contains PKG_NAME (when PKG_DIST_NAME does not), otherwise is may conflict with other packages. Downloaded files are cached in the distrib folder.

```suggestion
PKG_DIST_NAME = v$(PKG_VERS).$(PKG_EXT)
PKG_DIST_SITE = https://github.com/AlexxIT/go2rtc/archive
PKG_DIST_FILE = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
```